main.py

```diff
@@ -8,21 +8,16 @@
 
 database = db.DB("pugsey.db")
 logged_in_user = None
-user_info = None
-
+#user_info = None
 
 def login():
-    global user_info
-    global does_user_exist
     print("Login")
     email = cli_ui.ask_string("Email")
     password = cli_ui.ask_password("Password")
-
     does_user_exist = database.find(
         'Users', f'email = "{email}" AND password="{password}"')
-
     if len(does_user_exist) > 0:
-        user_info = user.User(does_user_exist[0][0], email, database)
+        login.user_info = user.User(does_user_exist[0][0], email, database)
```

Comment:
what's `login.user_info`? where is `login` defined?

Reply:
I found this way to use a variable defined in one function elsewhere